install-cockroachdb.md

```diff
@@ -178,7 +178,7 @@ $(document).ready(function(){
         <p>A C++ compiler that supports C++11. Note that GCC prior to 6.0 does not work due to <a href="https://gcc.gnu.org/bugzilla/show_bug.cgi?id=48891">this issue</a>. On Mac OS X, Xcode should suffice.</p>
       </li>
       <li>
-        <p>A <a href="http://golang.org/doc/code.html">Go environment</a> with a recent 64-bit version of the toolchain. The Makefile enforces the specific version required, as it is updated frequently. Be sure to set the <code class="highlighter-rouge">$GOPATH</code> and <code class="highlighter-rouge">$PATH</code> environment variables as described <a href="https://golang.org/doc/code.html#GOPATH">here</a>.</p>
+        <p>A <a href="http://golang.org/doc/code.html">Go environment</a> with a recent 64-bit version of the toolchain. The Makefile enforces the specific version required, as it is updated frequently.</p>
```

Comment:
It doesn't change *that* frequently. And we do require a specific version here (unlike with the C++ compiler, where we're more flexible), so we should just say we require Go 1.8 here.